blocks/core/ff_module/ff_module-tabs-component/ff_module-tabs-component.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var React = require('react');
+var TabsContent = require('../../ff_container/ff_container-tabs-content/ff_container-tabs-content.js');
+var TabsNavigation = require('..//ff_module-tabs-navigation-component/ff_module-tabs-navigation-component.js');
+var TabsControl = require('../ff_module-tabs/ff_module-tabs.js');
+var TabsValidator = require('../ff_module-tabs/_ff_module-tabs-control')();
+
+
+module.exports = React.createClass({
+	displayName: 'TabsComponent',
+	propTypes: {
+		modifier: React.PropTypes.string,
+		tabs: React.PropTypes.arrayOf(
+			React.PropTypes.shape({
+				state: React.PropTypes.string,
+				label: React.PropTypes.string.isRequired,
+				content: React.PropTypes.element.isRequired,
+				id: React.PropTypes.string.isRequired,
+				key: React.PropTypes.number.isRequired
+			})
+		).isRequired
+	},
+	componentDidMount: function() {
+		var options = {
```

Comment:
I'd add distinct target and trigger data-attributes here, to keep the functionality of the React component really separate from the vanilla one. Let me know if you're not sure what I mean :)